### PR TITLE
feat: [OS-55] Send syslog events to central server before & after build / dismantles

### PR DESCRIPTION
**Pull request by sartaj10**
### Description

Send syslog events to central server before & after build / dismantles

### Checklist

- [ ] PR Title format: `type: [OS-XXX] Short description` 
    - `type` is one of: 'build', 'ci', 'chore',  'docs',  'feat', 'fix',  'perf', 'refactor', 'revert', 'style', 'test' 
    - `OS-XXX` refers to a Jira issue. 
    - If this is a breaking change prefix the description with "BREAKING CHANGE:"
- [ ] There is a related Jira issue for this pull request
- [ ] Description should be a meaningful summary of the changes you are proposing
- [ ] For breaking changes prefix the title with `"BREAKING CHANGE:"` and include details in the description
- [ ] This PR includes tests related to these changes, or existing tests provide coverage
- [ ] This PR has updated documentation as appropriate
- [ ] `mvn clean package` runs successfully

### Notes
- Once approved, use the **squash and merge** option.
- Refer to the [development notes](https://github.com/esnet/oscars/blob/master/docs/development_notes.md#pull-requests) for more details.

### CODE REVIEW

**Review by haniotak — CHANGES_REQUESTED** [review]
will need to add some documentation to the effect that we will now need logback.xml files to be deployed

**Comment by sartaj10** [comment]
@haniotak Where do you want me to add that?

**Comment by sartaj10** [comment]
Also, do we need to add this stuff that we did to test syslog?

```
brew install rsyslog
brew services start rsyslog
mkdir /usr/local/var/run/

vi /usr/local/etc/rsyslog.conf
brew services restart rsyslog
/usr/local/var/log/rsyslogd.log

#rsyslog.conf contents:
$ModLoad imudp
$UDPServerRun 10514

# Provides TCP syslog reception
$ModLoad imtcp
$InputTCPServerRun 10514

*.*                /usr/local/var/log/rsyslogd.log
```

File: backend/config/logback.xml [hunk 1]
```diff
@@ -3,14 +3,56 @@
...
+        <encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
+            <Pattern>
+                %d %clr(%-5p) %clr(%-35logger{5}){cyan} - %-60msg%n
+            </Pattern>
+        </encoder>
+    </appender>
+
+    <appender name="SYSLOG" class="ch.qos.logback.classic.net.SyslogAppender">
+        <!--Need to change this-->
+        <syslogHost>127.0.0.1</syslogHost>
```
Comment (sartaj10) [inline backend/config/logback.xml:25]: Regarding this, do we need to change this to something else? If so, what should it be? Same, for pss/config/logback.xml

Comment (haniotak) [reply backend/config/logback.xml:25]: In the committed files this should not change. Do add the instructions on how to do localhost testing. For the deployed files we (you :) will need to coordinate with networking team to figure out where to send syslog events, and explain to them what syslog entries our operators / engineers will be seeing.

**Pull request merged**